context/app/static/js/pages/LineUpPage/LineUpPage.jsx

```diff
@@ -11,9 +11,11 @@ import Paper from '@material-ui/core/Paper';
 import SectionHeader from 'js/shared-styles/sections/SectionHeader';
 import metadataFieldTypes from 'metadata-field-types';
 
-function LineUpPage(props) {
-  const { entities } = props;
-  const firstRow = entities[0];
+function LineUpPage({ entities }) {
+  const cleanEntities = entities.map((entity) =>
+    Object.fromEntries(Object.entries(entity).filter((pair) => pair[1] !== null)),
```

Comment:
```suggestion
    Object.fromEntries(Object.entries(entity).filter(([,v]) => v !== null)),
```
A little more clear to me, but not a big deal.